.github/workflows/automated_docker_release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release image for local development environment
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+env:
+  IMAGE_NAME: ${{ github.repository }}/${{ github.event.repository.name }}
+
+jobs:
+  build:
+    name: Test
```

Comment:
This doesn't seem like the right name for this job

Reply:
Yes, I'll fix it